### ISSUE TITLE
Deploy problems & web.config

### ISSUE THREAD

**Issue by aarongustafson** [Freizeitler/a-k-catalog#9]
We ended up fixing the issues on our end, but here are the notes form my colleague covering the issues we ran into:

> It requires bower and then run quite a few commands to make it run, including grunt serve (or at least the readme.md says that). He has tried to serve the content of _dist like this:

```
<handlers>
   <!-- Indicates that the index.js file is a node.js site to be handled by the iisnode module -->
   <add name="iisnode" path="_dist/index.html" verb="*" modules="iisnode"/>
</handlers>
```

But that’s an HTML file and not a node file (reason why we get a 500 error). I’ve updated this configuration on the server and now we can see something. The problem is that we don’t have access to the assets like json files. I think the easiest for him will be to put all the assets inside _dist.

He will also need to add the following to his web.config

```
<staticContent>
  <remove fileExtension=".json"/>
  <mimeMap fileExtension=".json" mimeType="application/json"/>
</staticContent>
```

Right after:

```
<httpErrors existingResponse="PassThrough" />
```

(around line 51)

**Comment by aarongustafson** [comment]
You can see the current deploy here: http://cocktail-o-clock-10k-10kapart2016.azurewebsites.net/

**Comment by Freizeitler** [comment]
Great! Thank you and your colleague for the support! I've just updated the web.config as suggested.

**Comment by aarongustafson** [comment]
I’m still getting the "Actualising… Sorry, I need a connection!" message. Is that expected?

**Comment by Freizeitler** [comment]
No, that would be a quite poor experience :D I loaded the app on my own server for testing and it works quite well: http://www.freizeitler.de/cocktail-app/
Maybe we should delete the entry and I'll set up a new repo with the app and its files in the master root. Seems to produce issues on an azure instance with this constellation...

**Comment by aarongustafson** [comment]
Can do. Do you need the description you entered?

**Comment by Freizeitler** [comment]
No. I'll write a new one ;) See you later
